templates/rhdm70-optaweb-externaldb.yaml

```diff
@@ -0,0 +1,210 @@
+  rhdm: "1.2"
+message: A new Business Automation Employee Rostering Optimizer application using an external DB has been created in your project. 
+parameters:
+- displayName: Application Name
+  description: The name for the application.
+  name: APPLICATION_NAME
+  value: myapp
+  required: true
+- displayName: Employee Rostering Optimizer Custom http Route Hostname
+  description: 'Custom hostname for http service route.  Leave blank for default hostname,
+    e.g.: <application-name>-optaweb-<project>.<default-domain-suffix>'
+  name: EXECUTION_SERVER_HOSTNAME_HTTP
+  value: ''
+  required: false
+- displayName: ImageStream Namespace
+  description: Namespace in which the ImageStreams for Red Hat Middleware images are installed. These ImageStreams are normally installed in the openshift namespace. You should only need to modify this if you've installed the ImageStreams in a different namespace/project.
+  name: IMAGE_STREAM_NAMESPACE
+  value: openshift
+  required: true
+- displayName: ImageStream Tag
+  description: A named pointer to an image in an image stream. Default is "1.0".
+  name: IMAGE_STREAM_TAG
+  value: "1.0"
+  required: false
+- displayName: "Datasource Minimum Pool Size"
+  description: "Sets xa-pool/min-pool-size for the configured datasource."
+  name: DB_MIN_POOL_SIZE
+  required: false
+- displayName: "Datasource Maximum Pool Size"
+  description: "Sets xa-pool/max-pool-size for the configured datasource."
+  name: DB_MAX_POOL_SIZE
+  required: false
+- displayName: "Datasource Transaction Isolation"
+  description: "Sets transaction-isolation for the configured datasource."
+  name: DB_TX_ISOLATION
+  required: false
+- displayName: optaweb rostering persistence Dialect
+  description: optaweb rostering server persistence dialect 
+  name: OPTAWEB_ROSTERING_PERSISTENCE_DIALECT
+  value: ''
+  required: false
+- displayName: optaweb rostering persistence Datasource
+  description: optaweb rostering server persistence datasource for persistence.xml 
+  name: OPTAWEB_ROSTERING_PERSISTENCE_DATASOURCE
+  value: ''
+  required: false
+- description: Database name
+  displayName: Database Name
+  name: RHDM_DATABASE
+  required: true
+- description: Database user name
+  displayName: Database Username
+  name: RHDM_USERNAME
+  required: true
+- description: Database user password
+  displayName: Database Password
+  name: RHDM_PASSWORD
+  required: true
+- description: Database driver, e.g. postgresql, mysql
+  displayName: Database driver
+  name: RHDM_DRIVER
+  required: true
+  value: mysql  
+- description: Database service host name
+  displayName: Database service host
+  name: RHDM_SERVICE_HOST
+  required: true
+- description: Database service port number
+  displayName: Database service port
+  name: RHDM_SERVICE_PORT
+  required: true
+- description: This parameter allows to configure datasource or xa-datasource
+  displayName: Database datasource / xa-datasource
+  name: NON_XA_DATASOURCE
+  value: "true"
```

Comment:
Add the memory amount that this service will need, for example:
https://github.com/jboss-container-images/rhdm-7-openshift-image/blob/rhdm71-dev/templates/rhdm71-full.yaml#L389

Reply:
Done.